src/cms/templates/generic_language_switcher.html

```diff
@@ -0,0 +1,34 @@
+
+<!--generic template for language switcher-->
+<!--please set variable "target" when including this code-->
+
+
+
+
+{% load content_filters %}
+
+<div id="language-switcher" class="flex flex-col z-10">
```

Comment:
```suggestion
<div id="language-switcher" class="flex flex-col z-10 mr-4">
```
Could you add a margin so there is a gap between the language switcher and the next element?
![Screenshot_2021-05-17 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/118516264-72f97480-b736-11eb-8a2e-6bd7c097bd90.png)
![Screenshot_2021-05-17 Integreat Redaktionssystem(1)](https://user-images.githubusercontent.com/16021269/118516271-74c33800-b736-11eb-9221-6ce1eaf68f68.png)